src/main/java/org/jenkinsci/plugins/workinghours/model/ExcludedDate.java

```diff
@@ -121,99 +87,119 @@ public Boolean shouldExclude(Calendar date) {
         return false;
     }
 
-    public static class DataContainer {
+    /**
+     * Minutes offset to the UTC time, indicates the base timezone of the excluded date.
+     * Default to UTC(UTC+0).
+     */
+    private int utcOffset = 0;
 
-        /**
-         * Minutes offset to the UTC time, indicates the base timezone of the excluded date.
-         * Default to UTC(UTC+0).
-         */
-        public int utcOffset = 0;
+    /**
+     * Name of the selected timezone.
+     */
+    private String timezone;
 
-        /**
-         * A type that indicates whether
-         */
-        public String type = DateType.TYPE_GREGORIAN;
+    /**
+     * A type that indicates whether
+     */
+    private String type = DateType.TYPE_GREGORIAN;
```

Comment:
I would probably be better to store this as DateType instead of String if possible